spec/requests/api/v1/listings/show_spec.rb

```diff
@@ -0,0 +1,37 @@
+describe 'GET api/v1/listings/:id', type: :request do
+  let(:email)                 { 'test@test.com' }
+  let(:first_name)            { 'usu' }
+  let(:last_name)             { 'ario' }
+  let!(:api_v1_listing_path)   { '/api/v1/listings/' }
```

Comment:
use the routes helpers. there is a helper which is called `api_v1_listings_path` already. Prefer using that. You can check the route and the route helper method name by doing a `rails routes` and checking in the first column